runtime/test/strategies/resolve-recipe-test.js

```diff
@@ -130,4 +130,31 @@ describe('resolve recipe', function() {
     recipe = await onlyResult(arc, ResolveRecipe, recipe);
     assert.isTrue(recipe.isResolved());
   });
+
+  it('map slots by slot connection tags', async () => {
+    let manifest = (await Manifest.parse(`
+      particle A in 'A.js'
+        A()
+        consume master #root
+          provide detail #info #detail
+      particle B in 'B.js'
+        B()
+        consume info
+      recipe
+        //slot 'id0' #root as s0
```

Comment:
Why's this commented out?

Reply:
just forgot to remove it - it is not needed, it is an external slot that is being returned by the slot-composer (mock).